src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/attr/Change.java

```diff
@@ -65,9 +70,9 @@
      */
     private String branch;
     /**
-     * Topic name.
+     * Topic.
      */
-    private String topic;
+    private Topic topic;
```

Comment:
This is dangerous as old serialized data will fail to load (for example old build records in Jenkins).
The old field unfortunately needs to stay.
You can mark it as `transient` so that new objects won't serialize the data. And add a `readResolve` method to convert the old field to the new one if possible.